tests/testthat/_snaps/shuffle_ard.md

```diff
@@ -432,3 +432,50 @@
       1 Overall ARM AGE      <NA>       p          0.05
       2 <NA>        AGE      continuous mean      75.1 
 
+# rename_ard_columns() works
```

Comment:
For snapshots, can we first convert to a data frame, so that any changes to the print method don't affect the tests? We can subset columns/rows  before snapshotting to keep the snap small.